packages/react-app/files/src/utils/routing.tsx

```diff
@@ -26,3 +26,56 @@ export function withResetScroll(WrappedComponent) {
     </ScrollToTop>
   );
 }
+
+
+interface NestedRoutes {
+  [key: string]: Route;
+}
+
+class Route {
+  private _path: string;
```

Comment:
personal preference here but why not just call this `path`.  If we require methods and properties be defined with public, protected or private we can let the tooling control access rather than appearing to control this by naming convention

Reply:
did you see the path getter on line 49?

that'll maybe eventually also return the resolved path with the tokens replaced

Reply:
Oh actually I need to replace this with the routing lib I made.